changelogs/unreleased/gh-8825-luajit-fixes.md

```diff
@@ -0,0 +1,8 @@
+## bugfix/luajit
+
+Backported patches from the vanilla LuaJIT trunk (gh-8825). The following issues
+were fixed as part of this activity:
+
+* Fixed the panic routine when `mprotect` fails to change flags for mcode area.
+* Fix handling of instable types in TNEW/TDUP load forwarding.
+* Handle table unsinking in the presence of `IRFL_TAB_NOMM`.
```

Comment:
"Handled", I suppose? Bullet-list should be grammar-consistent.

Reply:
The same with "Fix" above. _**Fixed**_, thanks ;)